docs/logs/settings.md

```diff
@@ -0,0 +1,84 @@
+title: Sematext Logs Settings
+description: Cloud based SaaS / On-premises logging as a service platform. Index machine data, search, correlate and investigate. Build in reports and dashboards with monitoring and alerting capabilities.
+
+The settings include:
+
+  - inviting new users to your App
+  - [authorizing public IPs to send data to your app via syslog](../logs/authorizing-ips-for-syslog)
+  - adjusting data retention time, daily volume, and limits
+  - [changing your app's plan](../logs/faq/#plans-prices)
+  - checking how much data is, or has been shipped to your App
+
+  ![Sematext Logs App Settings](https://sematext.com/docs/images/logs/logsene-app-settings.png)
+
+
+## Side Navigation
+
+The persistent region on the left that can be collapsed, is used to easily switch across monitoring, log management, transaction tracing, real user monitoring, and other user & team features. All side navigation tabs are selectable. If they have chevrons, it indicates that the top level section is collapsible and contains additional subitems. The subitems will let you explore all your Apps, or choose a particular App.
+
+Side navigation top level sections for infrastructure and application performance monitoring, log management, transaction tracing, and real user monitoring:
```

Comment:
Are we missing Experience below?  I'd add it even though it's not live yet.

Reply:
Added